doc/nrf/libraries/bootloader/index.rst

```diff
@@ -0,0 +1,11 @@
+.. _lib_bootloader:
+
+Bootloader libraries
+#####################################
```

Comment:
The header formatting line should match the length of the header title.

Reply:
yes, Thanks for pointing it out. Bartek just mentioned this.